src/features/ci-cd/models/release-additions.ts

```diff
@@ -0,0 +1,28 @@
+import type { AbstractSqlModel } from '@balena/abstract-sql-compiler';
+
+export const addToModel = (abstractSql: AbstractSqlModel) => {
```

Comment:
> How do you decide on making it a computed field

That's easy, it was decided while writing the spec for this feature :)
HQ: https://jel.ly.fish/8ea1c390-9a85-402d-978c-4d31dcb0d235

> I guess we could have built these as standard terms in the data model, right?

This could easily be a plain DB field, but then we would need to keep it in sync with the other fields. (In fact I originally implemented this with the DB field and then switched it to a computed term.)
But the best way to be sure about that, is to not have the field at all and just make it computed based on other fields (if you can). The downside on this is that POST/PATCH on the computed term need to be handled manually.

Atm `is_final = release_type == 'final'`, which might make the new field look unnecessary, but in this PR chain we will later drop the `release_type` field in favor of a `revision` field and change the definition to `is_final = revision IS NOT NULL`. At that point a separate `is_final` term is obviously useful, right?

Reply:
Got it. Thanks for the explanation.